app/helpers/application_helper.rb

```diff
@@ -13,8 +13,20 @@ def divide_in_chunks(arr, chunk_size)
     chunks
   end
 
-  def get_img_url(video)
-    video.img ? "/tmp/#{video.img}" : 'http://dummyimage.com/665x375/000000/00a2ff'
+  def get_large_cover_url(video)
+    video.large_cover.url || 'http://dummyimage.com/665x375/000000/00a2ff'
+  end
+
+  def get_small_cover_url(video)
+    video.small_cover.url || 'http://dummyimage.com/166x236/000000/00a2ff'
+  end
+
+  def get_video_link_url(video)
```

Comment:
It feels like this helper belongs on the model, since there is no logic here relative to the view template.